### DISCUSSION TITLE
Suggest implementing level mode?

### DISCUSSION THREAD

**Discussion by dalerkd** [plbrault/youre-the-os#211] [Ideas]
1. The process of alternating tension and relaxation

2. Stage victories at levels

Can bring more satisfaction

3. It seems like an infinite mode now

I know some people may like this, but it can provide a more popular and impressive way: for example, persisting for 5 minutes and 10 minutes each will bring a certain level of celebration animation

**Comment by plbrault** [comment]
Hello,

I am getting back into developing the game as I will have more free time to put on it over the next couple of months.

What I have in mind right now is to add a "Story mode" comprised of different stages. Each stage would correspond to a different era of computer hardware. In the first stage, we would have a single-core processor and little RAM, and there would be a delay when swapping pages because of the slowness of old hard drives. Processes would also stay blocked for longer in case of I/O events. As we would progress between stages, we would gain more cores and faster storage and I/O. Eventually, we would get to a stage with hyperthreading, which would impact the game as described [here](https://github.com/plbrault/youre-the-os/issues/62). In the late stages, I could see the concept of P-cores and E-cores being added, with different types of processes that are better suited for one type of core or the other.

Comments and ideas are welcome :slightly_smiling_face:
